docs/acp/user/develop-an-app/develop-an-app.md

```diff
@@ -126,17 +145,24 @@ Every app requires default endpoints for the App Registry Service. This service
 To be able to use your Mini Framework as an app, add a new Spryker module:
 
 ```bash
-docker/sdk cli composer require spryker/app-kernel spryker/message-broker-aws spryker/propel-encryption-behavior 
+docker/sdk cli composer require 'spryker/app-kernel:dev-feature/without-encryption\ as\ 0.1.0  spryker/message-broker-aws'
```

Comment:
```suggestion
docker/sdk cli composer require 'spryker/app-kernel:^0.1.1  spryker/message-broker-aws'
```

Reply:
the feature/without-encryption branch is merged and changed are released with the 0.1.1 tag